templates/zerver/emails/confirm_registration_mit.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
I have seen that this is the case in the existing templates, but shouldn't it be better with `<!DOCTYPE html>`?